README.md

```diff
@@ -1,4 +1,57 @@
-# devtools-protocol
+# DevTools Protocol
 
-This is the repository for devtools-protocol. You're welcome to
-[[contribute]](CONTRIBUTING.md)!
+This repository is for resources and discussion around aligning the developer tools protocol used by various browsers, JavaScript engines and debugging tools.
+
+## Goals
+
+- Help empower web developers to build great applications by supporting an ecosystem of diagnostics tools which work well across a variety of targets.
+- Reduce the need for target-specific "devtools protocol adapters" in order to simplify the experience for tool users and tool developers.
+- Provide a vendor-neutral forum to facilitate collaboration and consensus building between different protocol implementations and interested clients.
+- Reduce the engineering investment necessary to build compatible protocol implementations through sharing of test cases.
+
+[Contributions](CONTRIBUTING.md) from the community are welcome!
+
+## What is here in this repo
+
+Not much yet, just a [few issues](https://github.com/WICG/devtools-protocol/issues) tracking work we'd like to do and links to a few resources below.
+
+## Implementations
+Engines and browsers that all have a similar debugging protocol built-in:
+- [Chrome DevTools](https://chromedevtools.github.io/devtools-protocol/), also used by most other chromium-based browsers
+- [WebKit / Safari](https://github.com/WebKit/webkit/tree/master/Source/JavaScriptCore/inspector/protocol)
+- [Node.js](https://chromedevtools.github.io/devtools-protocol/v8/)
+- Firefox - [in development](https://groups.google.com/forum/#!msg/mozilla.dev.platform/4-4A8W-nP5g/Y9C9UkWTAAAJ)
+- Edge - in development (link forthcoming)
+
+Adapters that expose a common protocol:
+- [Microsoft Edge Diagnostics Adapter](https://github.com/Microsoft/edge-diagnostics-adapter) - to be replaced with native support (above).
+- [RemoteDebug iOS WebKit Adapter](https://github.com/RemoteDebug/remotedebug-ios-webkit-adapter)
+- See also [RemoteDebug list of adapters](https://remotedebug.org/adaptors/)
+
+See also [RemoteDebug Protocol Compatibility Tables](http://compatibility.remotedebug.org/)
+
+## Clients
+There are a variety of tools which target some or all of the above implementations of the protocol.  Here are a few resources:
+- [Chrome DevTools protocol clients](https://github.com/ChromeDevTools/awesome-chrome-devtools#chrome-devtools-protocol)
+- [VS Code debuggers](https://github.com/Microsoft/vscode-chrome-debug-core/blob/master/README.md)
+- [RemoteDebug integrations](https://remotedebug.org/integrations/)
```

Comment:
Let's add Sonarwhal (name updated) to this list, as well (it utilizes the EDA which is based on CDP): https://sonarwhal.com/ 

Thoughts on adding Lighthouse to the list, as well?

Reply:
Added Sonarwhal.  Note I wasn't trying to create a complete list here (mainly just link to other sites that were already trying to do that) since I didn't want to sign up myself for maintaining the list ;-).

From what @pavelfeldman said, it seems unlikely Lighthouse is a good example - it relies heavily on Chrome tracing which includes a lot of implementation details.  It seems pretty unlikely that Lighthouse would support an interoperable subset of the debug protocol, so maybe it's a counter example?

Reply:
@RByers RE not creating a complete list, that's fair :) I wanted Sonarwhal added as it shows MSFT is invested across a number of clients (I'm keen on showing we're already putting our money where our mouth is so folks trust our investment)

And fair enough on Lighthouse. Sounds like it is indeed an example of where we'll not interoperate, at least short to mid term (and seems even long-term). Thanks for clarifying!

Reply:
This repo contains a large list of tools built on top of CDP: https://github.com/ChromeDevTools/awesome-chrome-devtools, we should link that up (but it's very Chrome specific)

Reply:
Do we want to mention other non-webbby platforms that are using CDP and could benefit from this work? Examples:
- IOS: https://github.com/square/PonyDebugger
- Android: https://github.com/facebook/stetho
- Lua: https://github.com/muzuiget/mare
- Go: https://github.com/allada/go-debugger-devtools
- Alibaba Weex: https://github.com/weexteam/weex-devtool

Reply:
> This repo contains a large list of tools built on top of CDP: https://github.com/ChromeDevTools/awesome-chrome-devtools, we should link that up (but it's very Chrome specific)

Yep, that's already the first link in the list.

> Do we want to mention other non-webbby platforms that are using CDP and could benefit from this work?

I don't have an opinion on that (personally my priority is only the web, but it's fine for others here to have broader goals).  I'll land this as-is, then you can submit your own PR on top of this :-)